src/SDKs/EdgeGateway/Microsoft.Azure.Management.EdgeGateway/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure DataBox Edge Gateway Management Library")]
+[assembly: AssemblyDescription("Provides management functionality for Microsoft Azure DataBox Edge Gateway Management Resources.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
`AssemblyVersion` must also be `0.9.0.0`

Reply:
fixed